### DISCUSSION TITLE
Trouble to enter working server on synology

### DISCUSSION THREAD

**Discussion by geek111** [lloesche/valheim-server-docker#387] [General]
Hi I have problem to join to server who work on my synology docker container. 
If I put ip address my synology server and port then cant join to server.
I do exactly same what is in here https://github.com/lloesche/valheim-server-docker#synology-help
What im doing wrong?

**Comment by iloving** [comment]
I set mine up a long time ago so my memory is a bit fuzzy, but I recall having issues connecting until I set SERVER_PUBLIC=true, but that might have been for external access.

Can you post your ENV variables?  Does the docker log say anything?
